### PR TITLE
Add link for launcher.py and note MKL HW reqs for DLRS tutorial.

### PR DESCRIPTION
**Pull request by bd-dean**
This will address issues #560 & #559.

### CODE REVIEW

**Comment by mvincerx** [comment]
@bd-dean. Thanks for pushing this. I asked before if you were working on this but didn't receive a reply. I understand you're  busy in training, so I appreciate you multi-tasking. 

I prefer to use your PR, here, to address the issue of adding a note per @eadamsintel comments.  I'd like you to:
- Review the `warning` I added in GH-569, https://github.com/clearlinux/clear-linux-documentation/pull/569/files. 
- Then revise your `important` note to be more specific per GH-569. 
- *Add this text to your new commit message*: Closes #560, Closes #559

After you do that, I can close GH-569.

File: source/clear-linux/tutorials/dlrs/dlrs.rst [hunk 1]
```diff
@@ -185,14 +185,11 @@ Kubeflow multi-node benchmarks
...
 
-.. note::
-
-   To take advantage of the Intel® AVX-512 and VNNI functionality with the Deep
-   Learning Reference Stack, you must use the following hardware:
+.. warning::
 
-   * Intel® AVX-512 images require an Intel® Xeon® Scalable Platform
-   * VNNI requires a 2nd generation Intel® Xeon® Scalable Platform
+   If you choose the Intel® MKL-DNN or Intel® MKL-DNN-VNNI image, your platform must support the Intel® AVX-512 instruction set. Otherwise, an *illegal instruction* error may appear, and you won’t be able to complete this tutorial.
```
Comment (DnPlas) [inline source/clear-linux/tutorials/dlrs/dlrs.rst]: +1

**Comment by bd-dean** [comment]
@mvincerx I've reworded the note in the Kubernetes section to the wording you used in #569, and made it a "warning" not a "note."  Thanks!

File: source/clear-linux/tutorials/dlrs/dlrs.rst (hunk 1 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/dlrs/dlrs.rst]: +1

**Comment by mvincerx** [comment]
@DnPlas. Do you need to make available the new launcher.py? Or is that done already?

**Comment by mvincerx** [comment]
@DnPlas. Could you respond to my question above? Timing of release of this document depends on the availability of an up-to-date `launcher.py`.

**Comment by DnPlas** [comment]
@mvincerx launcher.py is the same as Kubeflow's v0.4.0 launcher.py (https://github.com/kubeflow/kubeflow/blob/v0.4-branch/tf-controller-examples/tf-cnn/launcher.py) no change is needed for that file at the moment. This PR can be merged.

**Comment by bd-dean** [comment]
Added a sentence to note that launcher.py is based on kubeflow v0.4 as noted by @DnPlas .  Should be good to go.

File: source/clear-linux/tutorials/dlrs/dlrs.rst [hunk 2]
```diff
@@ -243,7 +243,9 @@ Images
 
 You must add `launcher.py`_ to the Docker image to include the Deep
 Learning Reference Stack and put the benchmarks repo in the correct
-location. From the Docker image, run the following:
+location. Note that this tutorial uses Kubeflow v0.4.0, and cannot guarantee results if you use a different version.
```
Comment (DnPlas) [inline source/clear-linux/tutorials/dlrs/dlrs.rst]: +1

Comment (mvincerx) [reply source/clear-linux/tutorials/dlrs/dlrs.rst]: LGTM

**Pull request merged**